src/Makefile

```diff
@@ -64,6 +64,9 @@ FLAGS += -DUSE_POLLY
 ifeq ($(USE_POLLY_OPENMP),1)
 FLAGS += -fopenmp
 endif
+ifeq ($(USE_POLLY_ACC),1)
+LLVMLINK += -lPollyPPCG -Wl,-rpath=$(shell $(LLVM_CONFIG_HOST) --libdir),--no-as-needed -lGPURuntime -Wl,--as-needed
```

Comment:
I would suggest `jitlayers.cpp`. I'm not sure what you mean by "include in the appropriate context" though.

yes, macOS ld also has an `rpath` option, although it can have a subtly different meaning (in particular, if `libGPURuntime` has a correct install-name id – which can be fixed by `install_name_tool` if not – then the rpath shouldn't be necessary at all), and a slightly different syntax (for example, `@rpath` instead of `$ORIGIN`)

> requires us to provide its location in case of external LLVM builds

Makes sense, but the rpath being set here is `llvm-config --libdir`, which won't help that case anyways. This flag addition would be bad if it's using a global LLVM (adding `-rpath=/usr/lib` tends to confuse the linker in my experience), and counter-productive in the case where it's already in the local `usr/lib` (since that's already in the julia rpath, but this would add that path as an absolute location and potentially confusing the linker after installation).